### PR TITLE
Finish patient list command

### LINKED ISSUE #55: [DEV] As a user, I want to list all my patient details

### PR DESCRIPTION
**Pull request by radiankrisno**
### Summary

Resolves #55 

Add `list t/patient` command to the application.

### Additional

* Remove warning in LightTheme.css
* Update `asUnmodifiableSortedByIdObservableList()` in UniqueAbstractPersonList as before it is not sorted.

### Pictures / Gif

Capture

### Checks

- [X] Link PR to issue
- [X] Add Reviewer
- [x] Pass CI

### CODE REVIEW

File: src/main/java/gomedic/model/person/UniqueAbstractPersonList.java [hunk 1]
```diff
@@ -145,7 +145,12 @@ public void setPersons(List<? extends T> persons) {
      * Returns the backing list as an unmodifiable {@code ObservableList}.
      */
     public ObservableList<T> asUnmodifiableSortedByIdObservableList() {
-        return internalUnmodifiableList;
+        return internalUnmodifiableList
+            .sorted((abstractPerson, otherAbstractPerson) ->
+                abstractPerson.getId() == otherAbstractPerson.getId()
+                    ? 0
+                    : abstractPerson.getId().getIdNumber()
+                    < otherAbstractPerson.getId().getIdNumber() ? -1 : 1);
```
Comment (yyhangz) [inline src/main/java/gomedic/model/person/UniqueAbstractPersonList.java:153]: Sorry, I accidentally missed this out. Thanks! 😅

File: src/test/java/gomedic/storage/JsonAddressBookStorageTest.java [hunk 2]
```diff
@@ -112,6 +112,8 @@ public void readAndSaveAddressBook_allInOrder_success() throws Exception {
         original.addPatient(MAIN_PATIENT);
         jsonAddressBookStorage.saveAddressBook(original); // file path not specified
         readBack = jsonAddressBookStorage.readAddressBook().get(); // file path not specified
+        System.out.println(original.getPatientListSortedById());
+        System.out.println(new AddressBook(readBack).getPatientListSortedById());
```
[the remark this answers is not in the document]
Comment (simonjulianl) [reply src/test/java/gomedic/storage/JsonAddressBookStorageTest.java:116]: oh yeah lol

File: src/main/java/gomedic/model/person/UniqueAbstractPersonList.java (hunk 1 above)
Comment (radiankrisno) [reply src/main/java/gomedic/model/person/UniqueAbstractPersonList.java:153]: No worries!

File: src/test/java/gomedic/storage/JsonAddressBookStorageTest.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (radiankrisno) [reply src/test/java/gomedic/storage/JsonAddressBookStorageTest.java:116]: oh gg okay thanks

**Comment by simonjulianl** [comment]
oh wait lol I merge it alrdy sorry2

**Pull request merged**